tests/test-sources/plugins/by-name/none-ls/default.nix

```diff
@@ -159,7 +158,9 @@
                 enable = !(lib.elem sourceName disabled);
               }
               # Some sources have a package option with no default
```

Comment:
This comment is no longer strictly accurate.

It should probably mention throwing and `mkUnpackagedOption` instead?